editions/tw5.com/tiddlers/howtos/Generating Static Sites with TiddlyWiki.tid

```diff
@@ -26,7 +26,7 @@ The first RenderTiddlersCommand generates the HTML representations of individual
 
 It is also possible to produce a single HTML file that contains static representations of tiddlers, and uses standard HTML anchor links to jump between them.
 
-For example: <a href="alltiddlers.html">alltiddlers.html</a>
+For example: [[alltiddlers.html#HelloThere|http://tiddlywiki.com/alltiddlers.html#HelloThere]]
```

Comment:
It's a shame to lose the relative link here (and in the text-slicer tiddler above), as it allows things to work on the prerelease, and locally downloaded versions of the repo. I'd consider introducing a JS macro that produces a relative link with the text being the absolute value of the URL: ie  `<<relative-link "alltiddlers.html">>` would display as `http://tiddlywiki.com/alltiddlers.html` etc.